src/resolvers/projectResolver.ts

```diff
@@ -331,23 +331,40 @@ export class ProjectResolver {
   static addSearchQuery(
     query: SelectQueryBuilder<Project>,
     searchTerm?: string,
+    similarityThreshold = 0.4,
```

Comment:
It's still hardcoded.

Reply:
I meant something like this:

https://github.com/Giveth/impact-graph/blob/da2bd57612483ff99b6b17f5a66d904a7416fee5/src/services/recurringDonationStreamQueue.ts#L70

Reply:
I didn't want to pass it as a function param, so if you want to get it from config as above, it would be better to resolve it once globally somewhere at the top